test/test_quantized.py

```diff
@@ -144,6 +144,28 @@ def test_qrelu6(self, X):
                 self.assertEqual(qY, qY_hat,
                                  message="{} relu failed".format(name))
 
+    """Tests the correctness of the quantized::relu op."""
+    @given(X=hu.tensor(shapes=hu.array_shapes(1, 5, 1, 5),
+                       qparams=hu.qparams()))
+    def test_qtanh(self, X):
```

Comment:
Can you add testing of the qnnpack op as well? You can add the `qengine` as a hypothesis param, similar to what we did for qlinear/qconv.